xm-commons-config/HOW_TO_USE.md

```
@@ -0,0 +1,75 @@
+# order.yml — configuration refresh order
+
+Controls the order in which config files are delivered to a microservice
+(`onRefresh` / `onInit` / `refreshFinished`) — on startup and on config update events.
+
+## Location
+
+One file per tenant, in the tenant root:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)